services/core-api/app/api/verifiable_credentials/manager.py

```diff
@@ -461,17 +463,44 @@ def produce_untp_cc_map_payload(cls, did: str,
         # "tsf_operating_count"
         # "tsf_care_and_maintenance_count"
 
-        curr_appt = permit_amendment.permittee_appointments[0]
-        for pmt_appt in permit_amendment.permittee_appointments:
+        pmt_appts: List[MinePartyAppointment] = permit_amendment.permittee_appointments
+        curr_appt = pmt_appts[0]
+
+        permit_amendment_issue_date = permit_amendment.issue_date if isinstance(
+            permit_amendment.issue_date, date) else permit_amendment.issue_date.date()
+
+        for pmt_appt in pmt_appts:
+            pmt_appt_start_date = None
+            if not pmt_appt.start_date:
+                pmt_appt_start_date = datetime(1900)
+            elif isinstance(pmt_appt.start_date, date):
+                pmt_appt_start_date = pmt_appt.start_date
+            elif isinstance(pmt_appt.start_date, datetime):
+                pmt_appt_start_date = pmt_appt.start_date.date()
+            else:
+                print(type(pmt_appt.start_date))
+                print("WHAT?")
```

Comment:
😂 
Miiiiiight want something else for an error log message. A little more context for the rest of us.